### ISSUE TITLE
Clear and warm cache with asset indexing

### ISSUE THREAD

**Issue by kjbrum** [putyourlightson/craft-blitz#94]
I'm running into an issue where a job is added to the queue for what seems like every asset after updating the asset indexes.

I'm guessing this has something to do with having the following settings enabled?
```
'clearCacheAutomatically' => true,
'warmCacheAutomatically' => true,
```

Any tips or ideas on how to fix this?

**Comment by putyourlightson** [comment]
Thanks, I'll look into what can be done about this.

**Comment by putyourlightson** [comment]
Fixed in https://github.com/putyourlightson/craft-blitz/commit/99d049ad46a5fa91322aa1b7062091025359ff24 which will be released in version 2.1.0.
